### DISCUSSION TITLE
Microsoft.Toolkit.Diagnostics or CommunityToolkit.Diagnostics

### DISCUSSION THREAD

**Discussion by richardvaldiviesomacias** [CommunityToolkit/dotnet#234] [Q&A] [answered]
I have noticed that there are two NuGet packages that have Guard and  Throwhelper in them. Microsoft.Toolkit.Diagnostics and CommunityToolkit.Diagnostics. Which one is the right one?

**Comment by cleissonalves** [comment, marked as answer]
**Microsoft.Toolkit.Diagnostics 7.1.2** is the latest stable version and probably the last using this name.

**CommunityToolkit.Diagnostics 8.0.0** is the new version, currently in preview.

**Reply by richardvaldiviesomacias** [reply]
Thanks, @cleissonalves for your answer, so I should just go and use CommunityToolkit.Diagnostics then?

**Reply by Sergio0694** [reply]
Correct, the new packages going forward will be the `CommunityToolkit.*` ones 🙂

**Reply by richardvaldiviesomacias** [reply]
thanks @Sergio0694
